index.html.in

```diff
@@ -134,7 +134,7 @@
                                               <span data-text="Detect_Language" id="detectText">Detect language</span>
                                               <span id="detectedText"></span>
                                           </button>
-                                          <button type="button" class="btn btn-default dropdown-toggle btn-sm" data-toggle="dropdown">
+                                          <button type="button" id="srcLangSelector" class="btn btn-default dropdown-toggle btn-sm" data-toggle="dropdown">
```

Comment:
Well, this isn't really a selector per se. How about `#srcLanguagesDropdownTrigger`? Similar change below.